stdlib/importlib/abc.pyi

```diff
@@ -36,24 +36,29 @@ _Path: TypeAlias = bytes | str
 
 class Finder(metaclass=ABCMeta): ...
 
+class Loader(metaclass=ABCMeta):
+    def load_module(self, fullname: str) -> types.ModuleType: ...
+    def module_repr(self, module: types.ModuleType) -> str: ...
+    def create_module(self, spec: ModuleSpec) -> types.ModuleType | None: ...
+    # Not defined on the actual class for backwards-compatibility reasons,
+    # but expected in new code.
+    def exec_module(self, module: types.ModuleType) -> None: ...
```

Comment:
Given the comment here (this one doesn't actually exist on the class at runtime), maybe we should leave `exec_module` there in subclasses of `Loader`?

Reply:
I don't really know the machinery in `importlib`, so I am fine with both.

Reply:
Neither do I, so I'd prefer to play it safe and leave it in for now :)